lib/cache.js

```diff
@@ -0,0 +1,26 @@
+/** Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
Is this file used at all?

Reply:
Only [in one place](https://github.com/uber-workflow/probot-app-monorepo-sync/pull/10/files#diff-ecd13dbf53ffe880f4a4c5c62156ff29R76) at the moment. Leaving it around in case more caching is needed after I deploy this update. I don't fully yet know how fast this will go through our api rate limits when actually being used for our repos (this is why I currently have the [graphql rate limit logs](https://github.com/uber-workflow/probot-app-monorepo-sync/pull/10/files#diff-88028ef1e5615d5c30a53d1adab25fc6R42))